package.json

```diff
@@ -63,5 +64,11 @@
     "tslint": "^6.0.0",
     "typescript": "^3.8.3",
     "typescript-tslint-plugin": "^0.5.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint && npm run lint:scss",
+      "pre-push": "npm test"
```

Comment:
`npm test` could be pre commit too?

Reply:
Technically it could be. I split it to speed it up. Both linting and testing takes bit of a time. Testing will take more and more time so my intention is make it less intrusive for someone who commits often.

Reply:
Alternatively I could use just `pre-push` hook to do both linting and testing.